venues/fixtures-al/venues.json

```
@@ -1,378 +1,473 @@
+            "url": null,
+            "api_key": "",
+            "digital_pour_venue_id": "5d657d943527260064257abf",
+            "digital_pour_location_number": 1,
+            "untappd_location": null,
+            "untappd_theme": null,
+            "untappd_categories": "[]",
+            "taphunter_location": "",
+            "taphunter_excluded_lists": "[]",
+            "taplist_io_display_id": "",
+            "taplist_io_access_code": "",
+            "beermenus_categories": "[]",
+            "beermenus_slug": ""
+        }
+    },
+    {
+        "model": "venues.venueapiconfiguration",
+        "pk": 20,
+        "fields": {
+            "venue": 20,
+            "url": null,
+            "api_key": "",
+            "digital_pour_venue_id": "5d657d943527260064257abf",
+            "digital_pour_location_number": 2,
+            "untappd_location": null,
+            "untappd_theme": null,
+            "untappd_categories": "[]",
+            "taphunter_location": "",
+            "taphunter_excluded_lists": "[]",
+            "taplist_io_display_id": "",
+            "taplist_io_access_code": "",
+            "beermenus_categories": "[]",
+            "beermenus_slug": ""
+        }
+    },
+    {
+        "model": "venues.venueapiconfiguration",
+        "pk": 23,
+        "fields": {
+            "venue": 23,
+            "url": null,
+            "api_key": "",
+            "digital_pour_venue_id": "",
+            "digital_pour_location_number": null,
+            "untappd_location": 28327,
+            "untappd_theme": 108878,
+            "untappd_categories": "[\"Beer on Tap at Fractal Brewing Project\"]",
+            "taphunter_location": "",
+            "taphunter_excluded_lists": "[]",
+            "taplist_io_display_id": "",
+            "taplist_io_access_code": "",
+            "beermenus_categories": "[]",
+            "beermenus_slug": ""
+        }
+    },
+    {
+        "model": "venues.venueapiconfiguration",
+        "pk": 24,
+        "fields": {
+            "venue": 24,
+            "url": null,
+            "api_key": "",
+            "digital_pour_venue_id": "",
+            "digital_pour_location_number": null,
+            "untappd_location": 4632,
+            "untappd_theme": 14806,
+            "untappd_categories": "[\"Goat Island Brewing\"]",
+            "taphunter_location": "",
+            "taphunter_excluded_lists": "[]",
+            "taplist_io_display_id": "",
+            "taplist_io_access_code": "",
+            "beermenus_categories": "[]",
+            "beermenus_slug": ""
+        }
+    },
+    {
+        "model": "venues.venueapiconfiguration",
+        "pk": 25,
+        "fields": {
+            "venue": 25,
+            "url": null,
+            "api_key": "",
+            "digital_pour_venue_id": "",
+            "digital_pour_location_number": null,
+            "untappd_location": 26977,
+            "untappd_theme": 103534,
+            "untappd_categories": "[]",
```

```suggestion
            "untappd_categories": "[\"\"]",
```

This needs to have the empty string in the list in order for taps to load. With this, I see 11 taps.